wrappers/workflow_wrapper.py

```diff
@@ -67,12 +67,19 @@ def create_pull_request_review_in_current_sprint(issue, jira, issue_text, assign
                                                                      created_issue.key.encode('utf-8')))
 
 
-def update_existing_issue_in_current_sprint(task, issue, jira):
+def update_existing_issue_in_current_sprint(task, issue, jira, issue_text):
     if isinstance(jira, MyJiraWrapper):
         if task.fields.status.name.encode('utf-8') == 'Done':
             echo_skip("No Need of Update! Already Done!")
         else:
-            if issue.data['state'].encode('utf-8') == 'closed':
+            # change the title id it was update in source (e.g. github)
+            if issue.title.encode('utf-8') not in task.fields.summary.encode('utf-8'):
+                import pdb
+                pdb.set_trace()
```

Comment:
There is a missing `pdb` here

Reply:
thanks for identifying let me remove that